src/ui/components/TestSuite/views/TestMetadata/TestItemTree.module.css

```diff
@@ -1,18 +1,21 @@
 .Tree {
   display: flex;
   flex-direction: column;
-  gap: 0.75rem;
 }
 .Tree[data-nested] {
-  padding-left: 0.75rem;
 }
 
 .List {
   display: flex;
   flex-direction: column;
   gap: 0.5rem;
-  padding-left: 0.75rem;
+  margin-bottom: 1rem;
```

Comment:
This is subjective 😄  but I don't dig this margin change:
![Screen Shot 2023-05-16 at 8 59 39 AM](https://github.com/replayio/devtools/assets/29597/e3fc9978-b09a-456b-8b25-94f55d814c04)

Vertical space between our items isn't consistent, and there's a lot of space now between lists.

Reply:
Having larger spaces between lists is appropriate (in my subjective opinion) because they're the main thing you're scanning for.

It's not exactly the same as line height versus gaps between paragraphs but it's in the same ballpark.

But I take your point and there are consistencies we can probably find. Will merge this and file a separate task for margins.